tests/unit/test_static_dynamic_segments.py

```diff
@@ -21,10 +21,22 @@ def form_with_data(segment, *rules):
         "randomisation_percent",
     ]
 
+    model_formsets = [
```

Comment:
**Comment** https://github.com/wagtail/django-modelcluster v6.0 now needs the `formsets` to be declared